lib/matplotlib/collections.py

```diff
@@ -80,9 +80,7 @@ class Collection(artist.Artist, cm.ScalarMappable):
     # _offsets must be a Nx2 array!
     _offsets.shape = (0, 2)
     _transOffset = transforms.IdentityTransform()
-    _transforms = []
-
-
+    _transforms = np.empty((0, 3, 3))
```

Comment:
I think @pelson just means to add a normal comment explaining what the expected shape of `_transfroms` is and why?